doc/T2C_doc.md

```diff
@@ -1 +1,72 @@
-This is placeholder for documentation of T2C
\ No newline at end of file
+#Table To Conversation T2C#
+
+T2C is XLSX (MS Excell) based format for fast authoring of conversations in spreadsheets. Detailed description is in  [../examples/cz_app/xls/E_EN_T2C_authoring.xlsx](../examples/en_app/xls/E_EN_T2C_authoring.xlsx)
+
+
+##Examples of mapping T2C, XML and JSON##
+We provide here some examples to illustrate how WAW T2C, WAW XML and WA JSON map in particular use cases.
```

Comment:
This should be in more general document as it is related not only to T2C but also to WAW format. I would rename this file.

Reply:
done